app/service/family.py

```diff
@@ -222,6 +222,14 @@ def delete(
     if family is None:
         return None
 
+    # Validate family belongs to same org as current user.
```

Comment:
I wonder if we could put this (as a function) in https://github.com/climatepolicyradar/navigator-admin-backend/blob/feature/pdct-1065-user-org-not-checked-on-family-delete/app/service/authorisation.py for reuse?